Makefile

```diff
@@ -45,6 +45,15 @@ image := $(image)
 version := $(tag)
 version := $(version:v%=r%)
 
+release_branch := prod
+branch := $(shell git rev-parse --abbrev-ref HEAD)
+
+# if we are not on the release branch, then force supplementary to be set
+ifneq ($(branch),$(release_branch))
+    ifeq ($(supplementary),)
+        supplementary := $(shell echo $(branch) | tr -c -d \[A-z\]\[0-9\])
```

Comment:
I think you want `A-Za-z0-9`, otherwise brackets, backslashes, carets, underscores, and backquotes will be allowed.